src/ApiHandler.ts

```diff
@@ -501,40 +501,51 @@ export default class ApiHandler {
 		return api;
 	}
 
+	private isGenericCallArray(arrayToCheck: any): boolean {
+		return Array.isArray(arrayToCheck) && arrayToCheck.every((element: any) =>
+			element instanceof GenericCall
+		)
+	}
+
 	private parseGenericCall(genericCall: GenericCall): SanitizedCall {
 		const { args, sectionName, methodName, callIndex } = genericCall;
 
 		// Pull out the labelled args since we want the labels
 		let labelledArgs = JSON.parse(genericCall.toString()).args;
 
 		// If labelledArgs has a `call` or `calls` field, we then go look for it in
-		// the original args field where it will be an instance of a GenericCall
-		// which can then recursively sanitize
+		// the original args field where it will be an instance of a GenericCall or
+		// an array of GenericCall's. The GenericCall(s) are then recursively
+		// sanitized by this function.
+		// A possible pitfall to look out for here is if a call or array of calls
+		// does not have the parameter name `call` or `calls`.
 		if (labelledArgs.call !== undefined) {
-			const genericCallArg = args.find((a: any): boolean => a instanceof GenericCall);
-			if (genericCallArg instanceof GenericCall) {
+			const genericCallArg = args.find((argument: any): boolean => 
+				argument instanceof GenericCall
+			);
+
+			if (genericCallArg !== undefined) {
```

Comment:
Do we want to cover cases where `genericCallArg` is `null`? If so this can just be `if (genericCallArg)` but not sure what the requirement is here.